### ISSUE TITLE
Empty object after deserialization

### ISSUE THREAD

**Issue by brynzovskii** [sendinblue/APIv3-php-library#183]
By some reasone I get an object that has no contact ID once calling `createContactWithHttpInfo`. The ID returns correctly if I use `$content` without deserialization here https://github.com/sendinblue/APIv3-php-library/blob/229bfcd8fa76678526354078ca340c55ad506b48/lib/Api/ContactsApi.php#L754

**Comment by rajatsib** [comment]
Hi @brynzovskii 

Can you share a code snippet or script of how you are using this method?

**Comment by brynzovskii** [comment]
@rajatsib Hi, sorry for the late response.

Here's the script:

```
                $apiKey = 'xxxxxx';

                $credentials = SendinBlue\Client\Configuration::getDefaultConfiguration()->setApiKey('api-key', $apiKey);

                $apiInstance = new SendinBlue\Client\Api\ContactsApi(
                    new GuzzleHttp\Client(),
                    $credentials
                );

                $createContact = new \SendinBlue\Client\Model\CreateContact([
                    'email' => $request['email'],
                    'updateEnabled' => true,
                    'attributes' => [
                        'FIRSTNAME' => $request['firstname'].' ',
                        'LASTNAME' => $request['lastname'],
                        ....
                    ],
                    'listIds' => $request['list_ids']
                ]);

                try {
                    return $apiInstance->createContactWithHttpInfo($createContact);
                } catch (Exception $e) {
                    return [
                        'code' => $e->getCode(),
                        'message' => $e->getMessage()
                    ];
                }
```

In order to have proper response that I could use I had to change the response inside createContactWithHttpInfo manually  from:

```
            return [
                ObjectSerializer::deserialize($content, $returnType, []),
                $response->getStatusCode(),
                $response->getHeaders()
            ];
```

to:

```
            return [
                'client_id' => $content,
                'code' => $response->getStatusCode(),
            ];
```

**Comment by rajatsib** [comment]
Hi @brynzovskii 

Can you please use the method [createContact ](https://github.com/sendinblue/APIv3-php-library/blob/master/docs/Api/ContactsApi.md#createcontact) instead of createContactWithHttpInfo?
createContactWithHttpInfo is an internal method used by the lib.

**Comment by brynzovskii** [comment]
@rajatsib As per [docs](https://developers.sendinblue.com/reference/createcontact) I tried first the `createContact` method first. But the issue is that in case of contact update event the response is an empty array. Same link from docs says that status code from response can be used to check whether the contact was updated (204) or created (201). `createContactWithHttpInfo` logically does exactly what I need, returns http status code that I can use further. 

So in case of internal use of the mentioned method can I get the status code using `createContact `? Sending the code as response along with contact ID would useful.
